src/gcs.rs

```diff
@@ -640,23 +642,64 @@ mod tests {
     async fn get_private_object() {
         init();
 
+        // define the private object
         let private_bucket = test_bucket();
         let filename = "get_private_object.txt";
 
+        // generate test data of a few kB
+        let file_length = 4096;
```

Comment:
I actually did 769 bytes to make it something "not power of 2". So I dunno, 1234 bytes?

Reply:
1234 it is